scout/server/blueprints/variants/views.py

```diff
@@ -277,6 +277,16 @@ def cancer_variants(institute_id, case_name):
         page = int(request.args.get("page", 1))
         form = CancerFiltersForm(request.args)
 
+    # update status of case if visited for the first time
```

Comment:
You could move this function to the controllers and re-use it from several other functions: 
- snv cancer variants
- sv cancer variants
- snv variants
- sv variants
- str variants? (current situation is that you don't activate the case when you look at these)

Reply:
Good catch on the STR variants - I'll add that to this PR!